src/client/java/teammates/client/scripts/DataMigrationForFeedbackQuestionsToQuestions.java

```diff
@@ -0,0 +1,137 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import javax.jdo.Query;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Assumption;
+import teammates.logic.api.Logic;
+import teammates.storage.api.FeedbackQuestionsDb;
+import teammates.storage.api.QuestionsDb;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.FeedbackQuestion;
+
+/**
+ * Script to create a Question copy of old FeedbackQuestions.
+ * 
+ */
+public class DataMigrationForFeedbackQuestionsToQuestions extends RemoteApiClient {
+    
+    private final boolean isPreview = true;
+    
+    private enum ScriptTarget {
+        BY_TIME, BY_COURSE, ALL;
+    }
+    
+    ScriptTarget target = ScriptTarget.BY_TIME;
+    
+    private final int numDays = 100;
+    private final String courseId = "";
+    
+    
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForFeedbackQuestionsToQuestions().doOperationRemotely();
+    }
+
+    @Override
+    protected void doOperation() {
+        
+        Datastore.initialize();
+        List<FeedbackQuestion> feedbackQuestions;
+        if (target == ScriptTarget.BY_TIME) {
+            Calendar startCal = Calendar.getInstance();
+            startCal.add(Calendar.DAY_OF_YEAR, -1 * numDays);
+            
+            feedbackQuestions = getOldQuestionsSince(startCal.getTime());
+            
+        } else if (target == ScriptTarget.BY_COURSE) {
+            feedbackQuestions = getFeedbackQuestionEntitiesForCourse(courseId);
+            
+        } else if (target == ScriptTarget.ALL) {
+            feedbackQuestions = getAllOldQuestions();
+            
+        } else {
+            feedbackQuestions = null;
+            Assumption.fail("no target selected");
+        }
+        
+        List<FeedbackQuestionAttributes> feedbackQuestionAttributes =
```

Comment:
This name could be a little misleading as it could be referring to a single instance of `FeedbackQuestionAttributes`